### ISSUE TITLE
Add step to auth flow shows 11 items instead of 10

### ISSUE THREAD

**Issue by jblachly** [keycloak/keycloak#16378]
### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

core

### Describe the bug

In the authentication flow editor, clicking "Add step" purports to show 10 items at a time (1-10, 11-20, etc.)

However, 11 items are shown on each page. The first item is correct, and only the end index is incorrect. 

For example, note that "Deny access", the 11th item, is shown at the end of the first page and beginning of the second page.

![Screen Shot 2023-01-10 at 10 01 48 PM](https://user-images.githubusercontent.com/5987153/211708043-49ed893d-fb9f-479b-9297-64ed29b18036.png)

![Screen Shot 2023-01-10 at 10 02 09 PM](https://user-images.githubusercontent.com/5987153/211708089-57f4850a-fa47-4420-9ceb-fdfe21351456.png)

### Version

20.0.0

### Expected behavior

10 items per list page

### Actual behavior

11 items per list page

### How to Reproduce?

See description and screenshots above

**Comment by rmartinc** [comment]
@jblachly You are right, I checked that the endpoint `/realms/<realm>/authentication/authenticator-providers` returns all the providers and the UI manages the list. So I think it's a UI issue, retaging the area to UI.
